docs/source/tutorials/portfolio_optimization.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Before 'Introduction', maybe one sentence on what the whole tutorial is about. e.g. "In this tutorial, we show how financial portfolio optimisation can be cast as a quadratic unconstrained binary optimisation (QUBO) problem and then solved with the quantum approximate optimisation algorithm (QAOA). We compare QAOA with the solutions found by classical brute force search, and also show how modifying the QAOA 'mixer' can lead to improved results.</p><p> </p><p>typo: 'the market comprises of n shares' -&gt; 'the market comprise n shares' or 'the market contains n shares'</p><p>typo: In the introduction we say that Xiaoming's risk tolerance is p, but in the remainder it is q</p><p> </p><p>At the end you say, 'We can now solve this by QAOA as above'.. maybe remove 'as above'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>